paper.tex

```diff
@@ -2470,47 +2470,54 @@ \section{Cell lineages and ARGs}
+genome. Rather, genomes can be associated with, or ``tag'', events above
+(ancestral to) or below (descended from) them.
+For example, tagging the two genomes above a recombination event leads to the
+two node representation seen in
+Figs.~\ref{fig-simplification} and \ref{hudson_vs_bigARG},
+whereas tagging the genome
+below a recombination event leads to the more
+conventional graphs in Figs.~\ref{fig-ancestry-resolution} and \ref{fig-inferred-args}A,B.
+
+%% JK:  What does this add that is not in caption?
+% Thick black lines trace an ARG that describes the ancestry of the lower arm of
+% the four sampled chromosomes, and these lines pass through gARG nodes (green
+% chromosomal regions). A number of these nodes have only one parent and one
+% child in the ARG; these would often be omitted (taken as implicit) in a
+% minimally simplified gARG.
+
+The schematic illustrates an important point about the biological reality of polytomies.
+Three linages coalesce in the left-hand genome of individual $D_{10}$, but do
+so as the result of two successive bifurcations. This is \emph{necessarily} so,
+because the only known method of reproducing DNA is by (semi-conservative) duplication.
+Whether this polytomy is resolvable depends on the available mutational data.
+Mutations can occur along any cell lineages. For example,
+a mutation in the first cell division of $D_{10}$ could be shared between
+the two gametes produced by the cells in the left half of $D_{10}$
+but not shared by the right hand gamete.
+With enough mutations, each
 round of mitotic germline genome duplication within a single multicellular organism could
-potentially be distinguished.
-This implies that in ARGs that have been inferred from real data, polytomies represent a lack
-of mutational information (rather than marking a single ``multifurcation event'').
-The ability to represent polytomies, and conceptualise them as a (deliberately) imprecise statement about the order
-of coalescence within and between individuals,
-fits naturally into the gARG framework presented in this paper.
+in principle be distinguished.
+
+% Note that mutations can occur along any lineage in Fig.~\ref{fig-cell-lines}. This means, for example,
+% that a mutation in the first cell division of $D_{10}$ could be shared between
+% the two gametes produced by the cells in the left half of $D_{10}$ but not shared by
+% the right hand gamete. With a high enough mutation rate (or equivalently, a long enough genome), each
+% round of mitotic germline genome duplication within a single multicellular organism could
+% potentially be distinguished.
+% This implies that in ARGs that have been inferred from real data, polytomies represent a lack
+% of mutational information (rather than marking a single ``multifurcation event'').
+% The ability to represent polytomies, and conceptualise them as a (deliberately) imprecise statement about the order
```

Comment:
I do quite like these last 3 lines, to bring the topic back into the purview of the gARG and the paper in general. It might not be obvious that the gARG is a natural fit to this "world of imprecision", or that this is what we claim a polytomy in a gARG is representing?

I guess one point here is that if we try to infer an eARG from real data, but allow polytomies, we are implicitly "doing the gARG thing" of showing the outcome of the process, rather than the process itself. I was trying to imply that, but I didn't say it exactly.